lib/index.js

```diff
@@ -61,10 +61,11 @@ module.exports.parse = function(opt, callback) {
  * @param loadedXml {boolean} Whether the xml was loaded from the file-system.
  * @param callback {function} The callback function using Javascript PCS.
  */
+
 function _parseWithXml2js(xmlContent) {
-  return new Promise(function(resolve, reject) {
+  return new Promise(function (resolve, reject) {
```

Comment:
Request to update to use await instead of a `Promise`.

Reply:
@nsinghal12 since async/await is kind of wrapper over Promise. So in order to use async/await on any function, the function should have promise implementation in it. So here **_parseWithXml2js()** implementing promise. That's why we are able to use async/wait in **parse()** in line 22. So here we must use promise so that it can consume as async/await in **parse()**. Let me know your thought on this.

Reply:
Please read my comments and suggestions elsewhere in this review to understand more.